src/common/ManagedTelemetry/Telemetry/Events/IEvent.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Telemetry.Events
+{
+    public enum PartA_PrivTags
+           : ulong
+    {
+        /// <nodoc/>
+        None = 0,
+
+        /// <nodoc/>
+        ProductAndServicePerformance = 0x0000000001000000u,

```

Comment:
See my email about this.

Reply:
@enricogior I've updated the values to 0x0u, and moved them to TelemetryBase, so they can be overwritten.   Let  me know if this works, or you need anything else.  Also still need to validate the Preview Pane events before I close this PR.

Reply:
Sure, I'll test it and let you know if the events are getting generated as expected (I'll do a private build enabling the telemetry with the secret key).